.github/workflows/devbuild.yml

```diff
@@ -297,14 +298,18 @@ jobs:
             VERBOSE=1 USE_CLANG_TIDY=OFF \
             BUILD_QT=ON \
             CMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
-            CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3) -DMODMESH_USE_PYSIDE:BOOL=ON"
+            CMAKE_ARGS="-DPYTHON_EXECUTABLE=$(which python3)"
           make buildext VERBOSE=1
 
       - name: make pytest BUILD_QT=ON
         run: |
+          # PySide6 installed by pip will bundle with a prebuilt Qt,
```

Comment:
The comments clearly describe why we need to align the Qt library files and are important for future maintenance.  Thanks a lot.